packages/admin/admin/src/common/toolbar/ToolbarBreadcrumbs.tsx

```diff
@@ -0,0 +1,295 @@
+// TODO: Add theming and support for `sx` and `slotProps` when #1809 is merged: https://github.com/vivid-planet/comet/pull/1809
+import { ChevronDown, ChevronRight, ChevronUp } from "@comet/admin-icons";
+import {} from "@emotion/react";
+import { ButtonBase, css, ListItemText, Menu, MenuItem, styled, Theme, Typography } from "@mui/material";
+import React from "react";
+import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
+
+import { useStackApi } from "../../stack/Api";
+import { useObservedWidth } from "../../stack/breadcrumbs/utils"; // TODO: Update import when merged: https://github.com/vivid-planet/comet/pull/1755
+
+// TODO: Remove debug code when we have stories for the new Toolbar to simulate a large stack
+const __DEBUG__numberOfPages = 6;
+const __DEBUG__dummyStackPages = [
+    { title: "Lorem ipsum", url: "/lorem-ipsum" },
+    { title: "Foo Bar", url: "/foo-bar" },
+    { title: "Nested page", url: "/nested-page" },
+    { title: "Another page", url: "/another-page" },
+    { title: "More nested pages", url: "/more-nested-pages" },
+    { title: "And even more nested pages", url: "/and-even-more-nested-pages" },
+    { title: "Foo", url: "/foo" },
+    { title: "Lorem", url: "/lorem" },
+    { title: "More pages", url: "/more-pages" },
+];
+const __DEBUG__usedNumberOfStackPages = __DEBUG__dummyStackPages.slice(0, __DEBUG__numberOfPages);
+const __DEBUG__useDebugBreadcrumbData = false;
+
+const BreadcrumbLink = React.forwardRef<HTMLAnchorElement, RouterLinkProps>(({ to, ...rest }, ref) => (
+    <RouterLink innerRef={ref} to={to} {...rest} />
+));
+
+export type ToolbarBreadcrumbsProps = {
+    scopeIndicator?: React.ReactNode;
```

Comment:
Does this PR conflict with #1293?

Reply:
No but we might need to rethink how the breadcrumbs and scope-indicator will be used together in the new Toolbar. But I think we can do this after these things have been merged into the feature-branch and we implement the new Toolbar component. 

If that's ok with @thomasdax98

Reply:
Yes, I think that's good. I'll implement the new `Toolbar` this week.